### DISCUSSION TITLE
How to attach a file to copilot

### DISCUSSION THREAD

**Discussion by RichKame** [community/community#143714] [Mobile] [answered]
I am having trouble getting copilot to reference a file when asking a question to copilot in my iOS app.How do I specify the file?

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, version 1.183.1 (145298821)
Device: iPhone17,2 (iOS 18.1)
</details>

**Comment by AuroraEchos** [comment, marked as answer]
Hi, I'm not quite sure what your problem is, but here are some necessary steps to attach files when asking Copilot questions in iOS apps:

1. Use Copilot to ask questions in the context of the code base file.
2. Navigate to a file on GitHub and open it.
3. To ask questions about the entire file, click the Copilot icon in the upper right corner of the file view.
4. To ask questions about specific lines, select the lines and click the Copilot icon to the right of Select.

Of course, if you have any other questions, you can refer to [Ask Copilot in GitHub] (https://docs.github.com/en/copilot/using-github-copilot/asking-github-copilot-questions-in-github).
